eng/scripts/bomhelpers.ps1

```diff
@@ -323,8 +323,18 @@ function GeneratePatch($PatchInfo, [string]$BranchName, [string]$RemoteName, [st
     $cmdOutput = git fetch $RemoteName $releaseTag
 
     if ($LASTEXITCODE -ne 0) {
-      LogError "Could not restore the tags for release tag $releaseTag"
-      exit $LASTEXITCODE
+      # Fall back to old tag format: <artifactId>_<version>
+      $oldReleaseTag = "$($artifactId)_$($releaseVersion)"
```

Comment:
@jairmyree this is the fallback logic to the old tag format. 
CC: @srnagar @XiaofeiCao